.github/workflows/config/production-helplines.json

```diff
@@ -0,0 +1,3 @@
+[
+  "AS"
+]
```

Comment:
You have only `AS` here for testing, but eventually, we'll list them all, correct?

Reply:
Yes, sorry, should have explained in the PR it's the old chicken & egg problem, shouldn't merge the PR before it's tested but can't test the new workflow without it being merged

Reply:
Yes, sorry should have noted that on the PR

The old 'chicken & egg' problem that we shouldn't merge before we test but the new workflow can't be tested before it's merged